db/migrate/20220610055050_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] <b>REQUIRED:</b> And here: `t.references :author, foreign_key: { to_table: 'users' }`